modules/tensor_mechanics/examples/piston/piston_params.i

```diff
@@ -0,0 +1,130 @@
+## This example is documented on YouTube at:
+## https://www.youtube.com/watch?v=L9plLYkAbvQ
+## 
+## Additional files (e.g. the CAD model, results)
+## can be downloaded freely from Zenodo at:
+## https://doi.org/10.5281/zenodo.3886965
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  type = FileMesh #Read in mesh from file
+  file = piston_coarse.e
+[]
+
+# This is where mesh adaptivity magic happens:
+[./Adaptivity]
+  steps = 1
+  #initial_steps = 2
```

Comment:
delete this line if you do not need.

Reply:
See https://github.com/idaholab/moose/pull/18543/commits/98a239e402b3cf2b87f62288a0962570948c16af